nilearn/plotting/html_connectome.py

```diff
@@ -1,11 +1,13 @@
+import functools
 import json
+import warnings
 
 import numpy as np
 from scipy import sparse
-from .. import datasets
-from . import cm
+from nilearn import datasets
```

Comment:
we prefer relative imports

Reply:
why did this change? it seems unrelated

Reply:
We should switch that. PEP8 endorses Absolute imports.
Relative imports misbehave/break more and are less clear.

Reply:
Relative imports tend to break more for reasons that are not readily apparent, especially within IDEs. 
This one broke when I was doing this PR.

Reply:
this should be discussed in a separate issue then.

Reply:
Okay, anything else for this PR?

Reply:
nope, it looks good otherwise.